### PR TITLE
Add MultiVNC

### PR DESCRIPTION
**Pull request by bk138**
MultiVNC is a cross-platform Multicast-enabled VNC viewer based on LibVNCClient.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: net.christianbeier.MultiVNC.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+runtime: org.gnome.Platform # using GNOME since it contains libsecret already built
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: multivnc
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=host # for saving screenshots and logs
```
Comment (hfiguiere) [inline net.christianbeier.MultiVNC.yml:11]: that's very broad filesystem permissions

File: net.christianbeier.MultiVNC.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - /lib/debug
+      - /lib/wx
+      - /share/bakefile
+      - /share/aclocal
+    sources:
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.2.1/wxWidgets-3.2.2.1.tar.bz2
+        sha256: dffcb6be71296fff4b7f8840eb1b510178f57aa2eb236b20da41182009242c02
+  - name: multivnc
+    buildsystem: cmake
```
Comment (hfiguiere) [inline net.christianbeier.MultiVNC.yml:39]:
```suggestion
    buildsystem: cmake-ninja
```

**Comment by hfiguiere** [comment]
bot, build net.christianbeier.MultiVNC

File: net.christianbeier.MultiVNC.yml (hunk 2 above)
Comment (bk138) [reply net.christianbeier.MultiVNC.yml:39]: Oh, that wasn't in the docs. What's the intended benefit @hfiguiere ? - faster build speeds?

File: net.christianbeier.MultiVNC.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    sources:
+      - type: git
+        url: https://github.com/bk138/multivnc.git
+        tag: MultiVNC-0.6.0
+    cleanup:
+      - /lib/debug
```
[the remark this answers is not in the document]
Comment (bk138) [reply net.christianbeier.MultiVNC.yml:45]: I wasn't aware that flatpaks should ship debug symbols. We can have them in there, but can you point me at the docs?

**Comment by hfiguiere** [comment]
build fail because the appstream file is not installed.

**Comment by bk138** [comment]
> build fail because the appstream file is not installed.

Yep, forgot that one :roll_eyes: - should be fixed now.

**Comment by bk138** [comment]
bot, build net.christianbeier.MultiVNC

**Comment by bk138** [comment]
Ready for review round 2 @hfiguiere - I deem it more appropriate for the reviewer to resolve conversation, so I left them open.

**Pull request closed without merging**